src/integrationTest/resources/config/oracle/jdbcbase7.sql

```diff
@@ -0,0 +1,27 @@
+ALTER SESSION SET CONTAINER=XEPDB1;
+
+CREATE TABLE TEST.SAMPLE_TABLE
+(
+    ID    NUMBER,
+    DESCRIPTION VARCHAR2(128)
+);
+INSERT INTO TEST.SAMPLE_TABLE(ID, DESCRIPTION)
+VALUES (1, 'Test Oracle Download with SIARD CMD');
+
+
+CREATE FUNCTION TEST.TEST_FN(p_bool BOOLEAN) RETURN VARCHAR2 IS
+BEGIN
+    IF p_bool THEN
+        RETURN 'TRUE';
+    ELSE
+        RETURN 'FALSE';
+    END IF;
+END;
+/
+
+CREATE OR REPLACE FUNCTION APLKN_ARCHV_ANLUX.FK_ASLH_PRFG_VE_INHLT_MUSS (
```

Comment:
I think it would be nice if this function is created in the same schema: `CREATE OR REPLACE FUNCTION TEST.FK_ASLH_PRFG_VE_INHLT_MUSS`